hooks/queries/queryClient.ts

```diff
@@ -0,0 +1,3 @@
+import { QueryClient } from '@tanstack/react-query'
+
+export default new QueryClient()
```

Comment:
It's often useful to be able to directly access the global `queryClient`, so I just gave it its own file. Alternatively people can use `useQueryClient` and this could've been a private variable in the provider file.